visualization/app/codeCharta/ui/codeMap/threeViewer/threeSceneService.ts

```diff
@@ -29,7 +28,7 @@ export class ThreeSceneService implements CodeMapPreRenderServiceSubscriber {
 	public labels: Group
 	public edgeArrows: Group
 	public mapGeometry: Group
-	private lights: Group
+	private readonly lights: Group
```

Comment:
?

Reply:
My idea mentioned that I could make this readonly.